### ISSUE TITLE
[Arm64/Unix] new failure GitHub_13404.sh

### ISSUE THREAD

**Issue by sdmaclea** [dotnet/coreclr#13629]
The following test is failing on c440335be
JIT/Regression/JitBlue/GitHub_13404/GitHub_13404/GitHub_13404.sh

It was working on 3758e847988c

**Comment by sdmaclea** [comment]
@dotnet/arm64-contrib FYI

**Comment by sdmaclea** [comment]
@justinvp This regression was caused by #13613

**Comment by sdmaclea** [comment]
@janvorli @jkotas FYI.  I'll look at it but looks like 146 files were changed might take me a while to figure out what was broken.

**Comment by justinvp** [comment]
Sorry about that. Any idea why the failure wasn't showing up in CI? (Is this regression test not run as part of CI)? Note the majority of the 146 files changed are palsuite tests that were deleted that can be ignored while investigating this.

**Comment by janvorli** [comment]
@sdmaclea can you please point me to a PR with the test failing? I was quickly looking at a couple of recent PRs, but haven't seen it.

**Comment by sdmaclea** [comment]
@justinvp No problem.  Thanks for telling me about the obsolete tests.  I haven't looked at the test or the change yet.  It might be something simple.

@janvorli arm64 Ubuntu is not typically run on PRs.  I am seeing it on internal CI runs on the tip.  

The output is uninteresting
```
/home/vmjenkins/workspace/Dotnet/build_and_test/bin/tests/pr1/Tests/coreoverlay/corerun GitHub_13404.exe
FAIL
Expected: 100
Actual: 1
```

**Comment by mikedn** [comment]
13404 is a test I added in #13429. That's probably the source of the problem and not the PAL changes.

**Comment by mikedn** [comment]
Looks like a duplicate of #12893, I might as well fix that too.

**Comment by sdmaclea** [comment]
@mikedn Thanks.  My git bisect workflow doesn't check new tests.  

@justinvp My apologies.

**Comment by mikedn** [comment]
See #13659

**Comment by justinvp** [comment]
@sdmaclea No worries.

**Comment by sdmaclea** [comment]
Fixed by #13659
